commit e3347134faefe141e94453bd85a67622f9d6a75b
Author: Jacob Abel <thatcadguy@gmail.com>
Date:   Sun May 4 09:06:35 2014 -0700

    [intel-warning-fix][unknown-pragma]
    
    The Intel compiler complains about the OpenMP pragmas, so I wrapped them
    in ifdef's.

Index: oce/src/BRepMesh/BRepMesh_FastDiscret.cxx
===================================================================
--- oce.orig/src/BRepMesh/BRepMesh_FastDiscret.cxx
+++ oce/src/BRepMesh/BRepMesh_FastDiscret.cxx
@@ -259,7 +259,9 @@ void BRepMesh_FastDiscret::Perform(const
   #else
     // alternative parallelization not yet available
     int i, n = aFaces.size();
+#ifdef _OPENMP
 #pragma omp parallel for private(i)
+#endif
     for (i = 0; i < n; ++i)
       Process (aFaces[i]);
   #endif
Index: oce/src/BRepMesh/BRepMesh_IncrementalMesh.cxx
===================================================================
--- oce.orig/src/BRepMesh/BRepMesh_IncrementalMesh.cxx
+++ oce/src/BRepMesh/BRepMesh_IncrementalMesh.cxx
@@ -260,7 +260,9 @@ void BRepMesh_IncrementalMesh::Update(co
     tbb::parallel_for_each (aFaces.begin(), aFaces.end(), *myMesh.operator->());
   #else
     int i, n = aFaces.size();
+#ifdef _OPENMP
 #pragma omp parallel for private(i)
+#endif
     for (i = 0; i < n; ++i)
       myMesh->Process (aFaces[i]);
   #endif
